src/store/diagnostics.ts

```diff
@@ -97,12 +118,26 @@ export const actions = {
 
     rowToUpdate.mute = isMuted;
 
+    await dispatch(
+      'preferences/updatePreferencesData',
+      {
+        property: `diagnostics.mutedChecks.${ rowToUpdate.id }`,
+        value:    isMuted,
+      },
+      { root: true });
+
+    await dispatch(
+      'preferences/commitPreferences',
+      rootState.credentials.credentials as ServerState,
+      { root: true },
+    );
+
     commit('SET_DIAGNOSTICS', diagnostics);
   },
 };
 
 export const getters: GetterTree<DiagnosticsState, DiagnosticsState> = {
-  diagnostics(state: DiagnosticsState) {
+  diagnostics(state: DiagnosticsState, _getters) {
```

Comment:
Huh, did we need `_getters` for some reason? (It's probably fine to leave the change in, just curious.)

Reply:
No, I had a different solution for the getters in a previous commit, but I missed the unused param after cleaning up because of the `_`